package.json

```diff
@@ -18,7 +18,7 @@
     "doc": "lib/docs"
   },
   "scripts": {
-    "postinstall": "node common/scripts/install-run-rush.js update",
+    "ci-install": "node common/scripts/install-run-rush.js update",
```

Comment:
So renaming as ci-install and updating the github action (was previously travis) -- the internal official build servers manually do this.